web/wallet/Taskfile.yaml

```diff
@@ -19,4 +21,5 @@ tasks:
 
   serve:
     cmds:
-      - npm run dev
+      - npm install
```

Comment:
This command has side-effects. I would not recommend doing this as part of other steps without being clear

Reply:
Okay removed im not a web dev XD